python/delaunay.py

```diff
@@ -135,13 +137,19 @@ def distanceSquare(pt):
             self.boundaryEdges.add(e20)
 
             e01 = self.makeKey(e01[0], e01[1])
-            self.edge2Triangles[e01] = [0, ]
+            self.edge2Triangles[e01] = [
+                0,
+            ]
```

Comment:
Huh, I'm not sure about the original intention here: perhaps it was supposed to be a tuple? In any case, I don't think this makes it more clear.